scanner/k8s-assets/README.md

```diff
@@ -0,0 +1,62 @@
+# Kubernetes Assets Scanner
+
+The Kubernetes Assets Scanner is a tool designed to scan and collect information about services, pods, and containers running in a Kubernetes cluster. It processes the collected data and reports findings to a GraphQL API (presumably Heureka).
+
+## Prerequisites
+
+- Go 1.15 or later
+- Access to a Kubernetes cluster
+- Heureka system for reporting findings
```

Comment:
Are we expecting certain labels on the Kubernetes pods?

Reply:
We do but its configurable no?

Reply:
What do you mean? "expecting certain labels" should also be part of the README?

Reply:
Yes, this is referring back to the fact that we build support groups from certain labels / annotations on the kubernetes resources